### ISSUE TITLE
Creating a Organization as a New User

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#4565]
Working on another issue I found this problem. Suppose a new user is just signed up. Then they are able to create a collective using the home page; 

Using the profile menu: 

![image](https://user-images.githubusercontent.com/12435965/130307292-58a1d5c0-c0bc-4875-8af8-227c03de1f42.png)

Or reading through the homepage they might be able to click the following section; 

![image](https://user-images.githubusercontent.com/12435965/130307305-d0576279-3dd8-4b32-92cb-d67e41a84208.png)

But it seems to me that they have no way of creating an organization. Only once they create a collective they see; 

![image](https://user-images.githubusercontent.com/12435965/130307394-31868d61-3e53-4bd6-b45d-8296e5ac4fc3.png)

in their profile menu. I am wondering if this is a significant limitation, in the sense that we don't give the user the ability to quickly create an organization. Or am I missing something here? 🤔

**Comment by Betree** [comment]
It's an important issue, thanks for documenting it. I remember we surfaced that during demos and said we will adjust the design but we probably forgot to iterate on it.

**Comment by SudharakaP** [comment]
> It's an important issue, thanks for documenting it. I remember we surfaced that during demos and said we will adjust the design but we probably forgot to iterate on it.

You are welcome. Let us ping the design team here so they know about this one. 😺 

cc: @opencollective/design

**Comment by Betree** [comment]
@Memo-Es I like the fact that _Version 2_ has descriptions for each action, which can help to clear up the confusion new users are facing.

Ping @piamancini.

**Comment by SudharakaP** [comment]
Just my opinion, but the description of the `My Organizations` section maybe needs revisiting. I feel it might not give a clear idea why the user might want to create an organization. Collectives can also contribute to other collectives if I am not mistaken. 🤔

**Comment by znarf** [comment]
I like the one with description too.

I think we should do it, it's a problem for people not to be able to create Organizations from the menu.

**Comment by Betree** [comment]
> Just my opinion, but the description of the My Organizations section maybe needs revisiting. I feel it might not give a clear idea why the user might want to create an organization. Collectives can also contribute to other collectives if I am not mistaken. thinking

@SudharakaP Collectives can indeed contribute to other collectives under the same host using their budget (i.e. they can't use a credit card), but that's not their primary function. You don't create a collective to contribute, you create a collective to receive (and later disburse) funds.

Organizations on the other hand can have two main purposes:
1. Contribute to collectives (e.g. Facebook contribution to Babel)
2. Become a fiscal host (e.g. OSC)

I feel like the description properly covers (1). It's doesn't mention (2), but most of the time people create organizations solely for the first use case. So no blocker on my side.

@znarf If that's ok for you I can prioritize it for this week, since it's an `important`/`simple` issue

**Comment by SudharakaP** [comment]
> > Just my opinion, but the description of the My Organizations section maybe needs revisiting. I feel it might not give a clear idea why the user might want to create an organization. Collectives can also contribute to other collectives if I am not mistaken. thinking
> 
> @SudharakaP Collectives can indeed contribute to other collectives under the same host using their budget (i.e. they can't use a credit card), but that's not their primary function. You don't create a collective to contribute, you create a collective to receive (and later disburse) funds.
> 
> Organizations on the other hand can have two main purposes:
> 
>     1. Contribute to collectives (e.g. Facebook contribution to Babel)
> 
>     2. Become a fiscal host (e.g. OSC)
> 
> 
> I feel like the description properly covers (1). It's doesn't mention (2), but most of the time people create organizations solely for the first use case. So no blocker on my side.

Thanks for the explanation. That makes sense now. 🙂
